news/_posts/translation/2016-07-01-new-translation-system-prestashop-17.markdown

```diff
@@ -0,0 +1,266 @@
+
+Not all modules have a third level: Some modules only have strings for the back office, so in that case we’d use only the second level domain, e.g. *Modules.ImageSlider* for the "Image Slider" module, in order to keep it as simple as possible. The strings for these modules will be gathered under a simple file, and not a folder.
+
+
+![Install translation domain for PrestaShop 1.7](/assets/images/2016/07/domains_modules.png)
+
+In Crowdin, you will then have a mix of files and folders, like this:
+
+<img style="border: 1px solid #CCC; padding: 10px;" width="350" src="/assets/images/2016/06/modules_domains_crowdin.png">
+
+
+Some strings might be rather generic (like “Search” or “Settings updated”) and can be shared with other modules or pages. In that case, we’ll use the *Admin*. or *Shop*. domains to avoid unnecessary duplicates. It means that in some modules folder you will see some *Admin.Actions* or *Shop.Theme* domains.
+
+
+###  Shop
+
+For the front office, it’s getting a little bit more complex. Any string related to the front office goes into a *Shop* domain. That’s the first level. Then it’s split in various sections, mostly functional. Here’s how it works for the second level:
+
+| Domain | Strings |
+| ---------- | -------- |
+| Emails | The emails sent from the shop to a customer (order confirmation, account creation, etc.).|
+| PDFs | The PDFs sent to a customer (invoice, delivery slip, credit slip, etc.).|
+| Theme | The strings attached to the default theme “Classic”. |
+| Demo | The content from the demo products and demo pages (“About us”, “Terms and conditions of use”, etc.). If you remove the demo content, you shouldn’t need any of these strings. |
+| Notifications | The warning, error or success messages that can appear in the shop. |
+| Forms | The forms available in the shop (“Contact us” page, addresses, etc.).|
+
+
+
+Each of this domain can be further divided to provide even more context.
+
+1. The **Theme** section is divided into 4 additional sections:
+  * Catalog: all the strings needed to display your catalog (product page, categories, etc.).
+  * Customer Account: anything related to the management of a customer account and the orders.
+  * Checkout: everything related to the act of buying - i.e. if you’re in catalog mode, you shouldn’t need these strings.
+  * Actions: all the call-to-actions, buttons or links that you find on the shop and that aren’t specific to a page or context.
+
+2. The **Demo** section has only two sections:
+  * Catalog: content for demo product and categories (descriptions).
+  * Pages: content inside the demo pages (formerly known as CMS pages).
+
+3. **Notifications** are split according to their level of alert: error, warning or success. They are the messages showing up on your shop.
+
+4. As for the **Form** sections, you’ll have
+  * the form labels,
+  * its corresponding errors (distinct from the theme errors, this one are specific to the form and will display inline),
```

Comment:
distinct from the theme errors, these one are specific to the form and will display inline